src/Tools/dotnet-dump/Commands/ExitCommand.cs

```diff
@@ -12,8 +12,11 @@ namespace Microsoft.Diagnostics.Tools.Dump
     [CommandAlias(Name = "quit")]
     public class ExitCommand : CommandBase
     {
+        static ExitCommand() { Exited = false; }
+        static public bool Exited { get; set; }
         public override void Invoke()
         {
+            Exited = true;
```

Comment:
I think a better solution is to add a public property called "Exited" to the ConsoleProvider class and have this property return m_shutdown which is field that gets set to true with Console.Exit() is called.

And above in Analyzer.cs you can check _consoleProvider.Exited() instead of your new ExitCommand property

Reply:
I named the property `Shutdown` to be consistent with its internal variable name.... Done